woodwork/tests/accessor/test_statistics.py

```diff
@@ -1250,3 +1250,19 @@ def test_box_plot_quantiles_errors(outliers_df):
     error = "quantiles must be a dictionary."
     with pytest.raises(TypeError, match=error):
         series.ww.box_plot_dict(quantiles=1)
+
+
+def test_box_plot_optional_return_values(outliers_df):
```

Comment:
Do you think we should add an explicit condition to this test to check that the extra keys are included when `include_indices_and_values=True`? We sort of implicitly test that in other tests, but if the purpose of this is to check that the right keys are returned maybe it would be good to check both the `True` and the `False` case.

Reply:
Added checks with the explicit `include_indices_and_values=True`!